distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -415,13 +406,11 @@ private void cleanupCometDClients() {
     public void testFederatedQueryByWildCardSearchPhrase() throws Exception {
         String queryUrl = OPENSEARCH_PATH.getUrl() + "?q=*&format=xml&src=" + OPENSEARCH_SOURCE_ID;
 
-        // @formatter:off
```

Comment:
❗️ If you're going to remove these blocks, you should run the formatter over the class...and let it update the 400+ lines that will be impacted.

And probably justify that choice. Personally, I would rather we get away from protected code blocks from the formatter so you won't have to work hard to justify to _me_; others may disagree.